core/tauri-runtime/src/webview.rs

```diff
@@ -48,6 +49,9 @@ impl From<&WindowConfig> for WebviewAttributes {
     if let Some(effects) = &config.window_effects {
       builder = builder.window_effects(effects.clone());
     }
+    if config.incognito {
+      builder = builder.incognito(true);
+    }
```

Comment:
move this up to line 39 and change it to
```suggestion
    builder = builder.incognito(config.incognito);
```

Reply:
Should be done!